addons/gamification/tests/test_performance.py

```diff
@@ -0,0 +1,30 @@
+from freezegun import freeze_time
+
+from odoo.addons.web.tests.test_login import TestWebLogin
+from odoo.tests import tagged
+
+DATE_TO_EPOCH_TEST_1 = '2023-04-27 13:01'
+DATE_TO_EPOCH_TEST_2 = '2023-04-27 17:11'
+assert DATE_TO_EPOCH_TEST_1 != DATE_TO_EPOCH_TEST_2
+
+
+@tagged('post_install', '-at_install')
+class TestPerformance(TestWebLogin):
+    def test_web_login(self):
+        with freeze_time(DATE_TO_EPOCH_TEST_1, tick=True):
+            with self.assertQueryCount(182):  # com: 158 ; ent: 182 ; local gamification only: 125
+                try:
+                    super().test_web_login()
+                except AssertionError:
```

Comment:
why not just crashing ?

Reply:
it should not crash no ? (otherwise the original test in `TestWebLogin` will also crash, not sure to unserstand why the try / except is required sorry)

Reply:
can't we just remove `@tagged('post_install', '-at_install')` so ?